### ISSUE TITLE
AMD-SNP: "Multi-socket" configuration supported?

### ISSUE THREAD

**Issue by viniul** [AMDESE/AMDSEV#119]
I am considering a multi-socket setup involving two AMD EPYC CPUs. 
Does AMD-SNP support such a multi-socket setup? If so, is it possible to assign cores of both CPUs to one virtual machine? Is there any documentation that gives more details on setting up a multi-socket system for AMD-SNP? 
Thanks!

**Comment by tlendacky** [comment]
Yes, SNP is supported on a multi-socket system. There really isn't much difference between a single-socket and two-socket system as far as general SNP support is concerned.

**Comment by wdsun1008** [comment]
> Yes, SNP is supported on a multi-socket system. There really isn't much difference between a single-socket and two-socket system as far as general SNP support is concerned.

Multi socket system report is signed by which VCEK?  sev-tool  only uses so0.
